package.json

```diff
@@ -29,50 +29,39 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch:lib": "watch 'npm run lint && npm test' ./lib"
   },
   "dependencies": {
```

Comment:
What to do with the CNR corresponding to this PR? Once option could be using a general message like this:

```
- Fix: updating dependencies due to known vulnerabilities in the previous ones
```

but I think it worth the time to include the detail. Something like:

```
- Fix: updating dependencies due to known vulnerabilities in the previous ones
      async: 1.5.2 -> 2.6.1
      body-parser: ~1.11.0 -> ~1.18.3
      ...
```

and so on.

What do you think?

Reply:
I like your proposal :)

Reply:
Fixed with 659219729e67f83b508555b3b36ddf3466dddf8a